examples/gno.land/p/sys/users/users.gno

```diff
@@ -0,0 +1,43 @@
+package users
+
+import (
+	"std"
+)
+
+type UserLookup struct {
+	Name  string
+	Addr  std.Address
+	Alias bool // lookup was found via an alias
+}
+
+// NewUserLookup returns a new UserLookup struct
+func NewUserLookup(name string, addr std.Address, alias bool) UserLookup {
```

Comment:
Remove this constructor.

Reply:
[f964e6f](https://github.com/gnolang/gno/pull/3166/commits/f964e6f602d3c135b6d6a1538bc05f9d8aafb88b) & [03b246e](https://github.com/gnolang/gno/pull/3166/commits/03b246e63083a6ec19495b01debe89b7b7d32336)